src/validation/custom.validation.ts

```diff
@@ -295,3 +295,31 @@ const checkCorporatesAddress = (trust: trustType.Trust, addressMaxLength: number
     }
   }
 };
+
+export const validateEmail = (email: string, maxLength: number) => {
+  checkEmailIsPresent(email.trim());
+  checkIsWithinLengthLimit(email.trim(), maxLength);
```

Comment:
I would not trim it for every call, just create single variable and pass it to the three functions, furthermore we do not need to return true from those function, their values is not used

Reply:
I'm not sure the trim call really matters either way. We cannot remove return true as the validation that calls this requires it. You can verify this by running the unit tests without returning true.

Reply:
Updated the trim to use a variable.

Reply:
`return true` should be removed from `checkEmailIsPresent`, `checkIsWithinLengthLimit` and `checkCorrectIsFormat`, it is not used, we need to `return true` for the main email custom validator,  if successfu,l as expected by `express-validator`.

Reply:
Updated this.